Kuzzle.Tests/API/Controllers/RealtimeControllerTest.cs

```diff
@@ -88,8 +88,9 @@ public async void SubscribeAsyncTest(SubscribeOptions options)
             {
                 expectedQuery.Merge(JObject.FromObject(options));
             }
+            Mock<RealtimeController.NotificationHandler> mockNotifHand = new Mock<RealtimeController.NotificationHandler>();
```

Comment:
(nitpicking) I'm not fond of truncated variable names, to me `notificationHandlerMock` is far more explicit and easier to read.